prod/onChainData128.8-validator1.yaml

```diff
@@ -290,3 +290,87 @@ EUROe-USD:
         params:
           id: euroe-stablecoin
           currency: USD
+
+
+WRBTC-DOC:
```

Comment:
We don't need this pair.
Considering the other feeds included, we can use this entry for pair WRBTC-BITP from Sovryn.